### PR TITLE
Master

### PR DESCRIPTION
**Pull request by damianvila**
I added a couple new options to the plugin (similar to existing YouTube features): an option to add an "Advertisement" sign, an option to add a "Video will start in: " sign if allowSkip is not being used and corresponding lang strings. I also corrected how loadingSpinner was hid to use native functionality and showed it, hid poster an play button on preroll exit so user don't see the play button and see the loading spinner to get the correct feedback (that the video is loading). And updated the Readme file.

### CODE REVIEW

File: src/videojs-preroll.js [hunk 1]
```diff
@@ -9,22 +9,26 @@
     src : '', //Advertisement source, can also be an object like {src:"file.mp4",type:"video/mp4"}
     href : '', //Advertised url
     target: '_blank', //Target to open the ad url in
-    allowSkip: true, //Allow skipping of the ad after a certain period
+    allowSkip: false, //Allow skipping of the ad after a certain period
```
Comment (dirkjanm) [inline src/videojs-preroll.js]: Could you set this back to true? This is also documented as the default

**Comment by dirkjanm** [comment]
Looks nice, will test and merge when I have time :) Thanks!

File: src/videojs-preroll.js (hunk 1 above)
Comment (damianvila) [inline src/videojs-preroll.js]: Oh! Yes, sorry! :stuck_out_tongue_closed_eyes:

**Comment by dirkjanm** [comment]
Merged, thanks for the nice additions!

**Pull request merged**